### PR TITLE
DNS-33 | Addition Features for creating loan record

### PR DESCRIPTION
**Pull request by omidiora**

### CODE REVIEW

**Comment by nextwebb** [comment]
@omidiora you have conflicts

**Comment by nextwebb** [comment]
@omidiora fix your codacy errors

File: api/urls.py [hunk 1]
```diff
@@ -1,13 +1,17 @@
...
+
+
+
 
 router = routers.DefaultRouter()
 router.register('user', UserViewSet)
 router.register('profile', ProfileViewSet)
 router.register('recovery', RecoveryViewSet)
 router.register('loan', LoanViewSet)
+router.register('delete/account', DeleteAccount)
```
Comment (nextwebb) [inline api/urls.py:14]: i think this feature should be within the Loan view class. you know we're actually tryna give the user the option to delete a loan record

**Comment by therealbigzeez** [comment]
Your codacy tests are still not passing

**Review by therealbigzeez — COMMENTED** [review]
Good to go, at last!

**Review by nextwebb — APPROVED** [review]
all tests passed.

**Pull request merged**